scrapers/AdultFilmIndex.yml

```diff
@@ -0,0 +1,75 @@
+name: AdultFilmIndex
+sceneByURL:
+  - action: scrapeJson
+    url:
+      - https://adultfilmindex.com/api/v1/stash/scene/
+    scraper: sceneScraper
+sceneByName:
+  action: scrapeJson
+  queryURL: https://adultfilmindex.com/api/v1/stash/scene_search/{}
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeJson
+  queryURL: "{url}"
+  scraper: sceneScraper
+sceneByFragment:
+  action: scrapeJson
+  queryURL: https://adultfilmindex.com/api/v1/stash/scene_fragment_search/{filename}/{oshash}
+  scraper: sceneScraper
+movieByURL:
+  - action: scrapeJson
+    url:
+      - https://adultfilmindex.com/movie/
+    scraper: movieScraper
+    queryURL: https://adultfilmindex.com/api/v1/stash/movie/{url}
+    queryURLReplace:
+      url:
+        - regex: '.+/movie/(.*)/(.*).+'
```

Comment:
althought it seems to work the regex seems a bit weird, eg
<https://regex101.com/r/LgAWOB/1>
if you only want to keep the movie id i would suggest something like this `.+/movie/([^/]+)/.+$`
instead <https://regex101.com/r/pLE432/1>

Reply:
You are right, that's much better regex, updated the PR